scheduler/models/tasks.py

```diff
@@ -86,6 +88,10 @@ def __hash__(self):
             f"{self.normalizer.id}-{self.raw_data.boefje_meta.id}-{self.raw_data.boefje_meta.organization}"
         ).hex()
 
+    @classmethod
+    def get_type(cls) -> str:
+        return "normalizer"
```

Comment:
I think this is not necessary if you assign `type` to a `ClassVar`

Reply:
Cool, haven't seen that yet